Gemfile

```diff
@@ -135,7 +135,7 @@ group :test, :development do
   gem 'rb-fsevent', require: RUBY_PLATFORM =~ /darwin/i ? 'rb-fsevent' : false
   gem 'rb-inotify', '~> 0.9', require: RUBY_PLATFORM =~ /linux/i ? 'rb-inotify' : false
   gem 'rspec-rails', require: false
-  gem 'shoulda', require: false
+  gem 'shoulda', '~> 3.6'
```

Comment:
small one, is there a reason you removed the require: false, I think we only need this in test no?

Reply:
hmmmmmm you are right! I can't remember the reason. I will update it accordingly 👍